developerportal/templates/header.html

```diff
@@ -64,6 +64,13 @@ <h4 class="mzp-c-menu-item-title">{{ page.title }}</h4>
                       </li>
                     {% endwith %}
                   {% endfor %}
+                  {% if BLOG_LINK %}
+                  <li class="mzp-c-menu-category">
+                      <a class="mzp-c-menu-title" href="{{ BLOG_LINK }}">
```

Comment:
@schalkneethling Do you think this link (which will be to Mozilla Hacks) should be a `nofollow`, too? (If so, i guess the footer links should be, too)